geopandas/io/tests/test_sql.py

```diff
@@ -483,6 +484,17 @@ def test_write_postgis_without_crs(self, engine_postgis, df_nybb):
         with pytest.warns(UserWarning, match="Could not parse CRS from the GeoDataF"):
             write_postgis(df_nybb, con=engine, name=table, if_exists="append")
 
+        sql = text(
+            "SELECT Find_SRID('{schema}', '{table}', '{geom_col}')"
+            " ORDER BY '{geom_col}' DESC;".format(
```

Comment:
is there a reason this order by is needed/  why we cannot reuse the same sql from above?

Could we instead just run `conn.execute(sql).fetchall()` below and check the srids for each row?